client/src/styles/modules/navbar.module.css

```diff
@@ -0,0 +1,34 @@
+.nav {
```

Comment:
Delete pls, also as a general rule of thumb, try to stick to tailwind's sizing options instead of using your own custom sizes like 120px. It just makes it easier for consistency across components when people can easily use `h-20` or something like that (Not a big deal tho)